searx/templates/simple/results.html

```diff
@@ -92,6 +92,25 @@ <h4 id="corrections-title">{{ _('Try searching for:') }}</h4>
     </div>
     {% endif %}
 
+    {% if has_misspelled_words %}
+    <form method="{{ method or 'POST' }}" action="{{ url_for('search') }}">
+      <input type="hidden" name="q" value="{{ spellcheck_query }}" >
+      {% for category in selected_categories %}
+      <input type="hidden" name="category_{{ category }}" value="1" >
+      {% endfor %}
+      <input type="hidden" name="pageno" value="{{ pageno+1 }}" >
+      <input type="hidden" name="language" value="{{ current_language }}" >
+      <input type="hidden" name="time_range" value="{{ time_range }}" >
+      <input type="hidden" name="safesearch" value="{{ safesearch }}" >
+      <input type="hidden" name="theme" value="{{ theme }}" >
+      {% if timeout_limit %}<input type="hidden" name="timeout_limit" value="{{ timeout_limit|e }}" >{% endif %}
+      {{- engine_data_form(engine_data) -}}
+      <span style="color: #ff7769; font-size: 18px"> Did you mean:
```

Comment:
Please don't hardcode style here - edit `https://github.com/searxng/searxng/blob/master/searx/templates/simple/preferences/query_in_title.html` with the CSS instead and run `static.build.commit` after.

Reply:
Apart from that I don't think we need a new color here, you can use an existing one of  https://github.com/searxng/searxng/blob/master/searx/static/themes/simple/src/less/definitions.less